Makefile

```diff
@@ -117,7 +119,7 @@ docker-build:
 
 	@test -s ./overlays/dev/manager_image_patch.yaml || bash -c 'echo -e ${MANAGER_IMAGE_PATCH} > ./overlays/dev/manager_image_patch.yaml'
 
-ifeq ($(TRAVIS),true)
+ifeq ($(ISIMGLOCAL),true)
```

Comment:
Suggest: `USELOCALIMG`

Reply:
Or, better USE_LOCAL_IMG

Reply:
I like `USE_LOCAL_IMG` variable name also. In this case, `TRAVIS` is a special environment variable to identify this is running in Travis so the chosen var will need to added to Travis.

Reply:
`ISIMGLOCAL` is set to `false` up above by default. I think it is a bit more generic than `TRAVIS` as we could be using a local image even outside of travis. WDYT? 

and I will update to `USE_LOCAL_IMG` to be more consistent with the rest of the const.